templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      emit "$name stop failed; still running.  Killing with SIGKILL"
+      trace "Killing $name (pid $pid) with SIGKILL"
+      kill -KILL $pid
+      sleep 1
+    fi
+    if status ; then
+      emit "$name stop failed; still running after SIGKILL."
```

Comment:
This check shouldn't be necessary. SIGKILL has the kernel wipe the process from the process table.